io/DerivativeWriter.cpp

```diff
@@ -161,9 +160,17 @@ void DerivativeWriter::write(const PointViewPtr data)
                     continue;
                 Matrix3d block = paddedDSM.block(r-1, c-1, 3, 3);
                 if (to.m_type == SLOPE_D8)
+                {
```

Comment:
May not be portable, but:
http://stackoverflow.com/questions/1727881/how-to-use-the-pi-constant-in-c